src/components/WorkerInterfaceValues.vue

```diff
@@ -110,6 +111,10 @@ export default class WorkerInterfaceValues extends Vue {
   @Prop({ default: "right", type: String })
   readonly tooltipPosition!: "left" | "right";
 
+  getTourStepId(id: string): string {
+    return `${toKebabCase(id)}-tourstep`;
+  }
+
```

Comment:
This function could be factorized and placed right after `toKebabCase`

Reply:
Yes, sorry, I should have done that earlier!